pkg/app/utils.go

```diff
@@ -94,3 +96,21 @@ func bastionDebugWorkloadSpec(ctx context.Context, name string, image string, na
 		},
 	}
 }
+
+// getVpcIdForRDSInstance gets the VPC_ID from env if set or from the default VPC
+func getVpcIdForRDSInstance(ctx context.Context, ec2Cli *ec2.EC2) (string, error) {
```

Comment:
Words in names that are initialisms or acronyms have a consistent case. (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)
```suggestion
func getVPCIDForRDSInstance(ctx context.Context, ec2Cli *ec2.EC2) (string, error) {
```